### PR TITLE
Search: update Instant Search description

### PR DESCRIPTION
**Pull request by bluefuton**
#### Changes proposed in this Pull Request:

We are planning to adjust our default overlay trigger to 'submit' in https://github.com/Automattic/jetpack/pull/25093. By default, this will mean that users will need to hit submit or Enter before the Instant Search modal appears.

The language we have previously used to describe Instant Search - "get search results as soon as they start typing" - is no longer strictly true. This PR suggests an alternative wording to reflect the new default behaviour. Not entirely sold on the current version so would appreciate some input! (cc @simonktnga8c)

#### Other information:

- [x] Have you written new tests for your changes, if applicable?
- [x] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion
pcNPJE-YF-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
Read the text and review the copy.

Screen Shot 2022-08-08 at 10 40 54

### CODE REVIEW

**Review by a8ck3n — DISMISSED** [review]
This looks good to me. What do you think about describing the Overlay as part of the copy under the second toggle though? I think this is one area that we aren't very clear on currently and it requires the user to experiment to discover that we are basically offering two separate UIs for the feature.

The changes address the concern of the previous copy being out of date though, so not a blocker!

File: projects/packages/search/src/dashboard/components/module-control/index.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const SEARCH_DESCRIPTION = __(
 	'jetpack-search-pkg'
 );
 const INSTANT_SEARCH_DESCRIPTION = __(
-	'Instant search will allow your visitors to get search results as soon as they start typing. If deactivated, Jetpack Search will still optimize your search results but visitors will have to submit a search query before seeing any results.',
+	'Instant search will allow your visitors to get our fastest search experience with rapid sorting and filtering.',
```
Comment (bluefuton) [inline projects/packages/search/src/dashboard/components/module-control/index.jsx]:
```suggestion
	'Instant search will allow your visitors to get our speediest search experience with lightning-fast sorting and filtering.',
```

**Comment by bluefuton** [comment]
> This looks good to me. What do you think about describing the Overlay as part of the copy under the second toggle though?

Good point @a8ck3n - I think that's worth addressing here. How about adding something like:

"Search results open in an overlay over the current page so a page reload is not required"

**Comment by a8ck3n** [comment]
> Good point @a8ck3n - I think that's worth addressing here. How about adding something like:
> 
> "Search results open in an overlay over the current page so a page reload is not required"

Maybe like this?

> Instant search uses a dynamic overlay allowing your visitors to get our speediest search experience with lightning-fast sorting and filtering without reloading the page.

Or maybe without that last part…

> Instant search uses a dynamic overlay allowing your visitors to get our speediest search experience with lightning-fast sorting and filtering.

Or another option…

> Instant search uses a dynamic overlay allowing your visitors to real-time results, sorting, and filtering without reloading the page.

Maybe I'm trying to hard to keep it all in one sentence!

**Comment by bluefuton** [comment]
Thanks @a8ck3n! Great suggestions. I really like the last one but will try to include "lightning fast" in it as we use that phrase elsewhere (e.g. in upsell).

File: projects/packages/search/src/dashboard/components/module-control/index.jsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const SEARCH_DESCRIPTION = __(
 	'jetpack-search-pkg'
 );
 const INSTANT_SEARCH_DESCRIPTION = __(
-	'Instant search will allow your visitors to get search results as soon as they start typing. If deactivated, Jetpack Search will still optimize your search results but visitors will have to submit a search query before seeing any results.',
+	'Instant search will allow your visitors to get our speediest search experience with lightning-fast sorting and filtering.',
```
Comment (bluefuton) [inline projects/packages/search/src/dashboard/components/module-control/index.jsx]:
```suggestion
	'Instant search uses a dynamic overlay for lightning-fast results, sorting, and filtering without reloading the page.',
```

File: projects/packages/search/src/dashboard/components/module-control/index.jsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ const SEARCH_DESCRIPTION = __(
 	'jetpack-search-pkg'
 );
 const INSTANT_SEARCH_DESCRIPTION = __(
-	'Instant search will allow your visitors to get search results as soon as they start typing. If deactivated, Jetpack Search will still optimize your search results but visitors will have to submit a search query before seeing any results.',
+	'Instant search uses a dynamic overlay for lightning-fast results, sorting, and filtering without reloading the page.',
```
Comment (kangzj) [inline projects/packages/search/src/dashboard/components/module-control/index.jsx]: I could be wrong, but `lightning-fast results` doesn't sound quite right to me. How about `lightning-fast searching` or `lightning-fast search`?

Comment (bluefuton) [inline projects/packages/search/src/dashboard/components/module-control/index.jsx]: Sounds good @kangzj!

```suggestion
	'Instant search uses a dynamic overlay for lightning-fast searching, sorting, and filtering without reloading the page.',
```

**Review by kangzj — APPROVED** [review]
This looks great to me 👍

**Comment by simonktnga8c** [comment]
That copy looks good to me @bluefuton 👍 (sorry I missed the ping!)

**Comment by bluefuton** [comment]
Thank you @simonktnga8c! 🙌

**Pull request merged**